internal/uploader.go

```diff
@@ -216,3 +255,41 @@ func (uploader *SplitStreamUploader) Clone() Uploader {
 		blockSize:  uploader.blockSize,
 	}
 }
+
+func (uploader *RegularUploader) ShowRemainingTime() {
+	defer uploader.waitGroup.Done()
+	startTime := time.Now()
+	for {
+		totalSizeInt, err := uploader.CompressedDataSize()
```

Comment:
Total backup size is unknown to the uploader. New data is being constantly feed to the uploader, see the rest of the code regarding backup-push.

This is not going to show real total backup size.